src/dbus/freedesktop_screensaver.rs

```diff
@@ -13,6 +13,7 @@ use zbus::{interface, Task};
 
 use super::Start;
 
+#[derive(Clone)]
```

Comment:
If you want this Cloned like that, you need to make the `counter` shared too, i.e. via `Arc<AtomicU32>` which you `fetch_add()` with `SeqCst` ordering just to be sure.